x-pack/plugins/apm/public/components/shared/backend_link.tsx

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Created components/shared/backend_link and components/shared/service_link at this level instead of in components/shared/Links/apm with the assumption that we'll be getting rid of most of the components in Links eventually (see #105914), but these are components that include the icon and special structure and formatting and do not belong with those that are mostly just wrapped `EuiLink`s.